speech/action.js

```diff
@@ -45,7 +93,7 @@ exports.sreconfig = function(data) {
 //    The second is for when a MathItem's render(), rerender() or convert() method is called.
 //
 exports.speechAction = {
-  simplfy: [
+  simplify: [
     STATE.ENRICHED + 1,
```

Comment:
I'd recommend defining a state for the render actions, for clarity.  Use `newState()` from `MathItem.js` (which you are require already anyway).  E.g.,

``` js
newState('SIMPLIFY', STATE.ENRICHED + 1);
newState('ROLE', 1000);
newState('DESCRIBE', STATE.ROLE + 1);
```

and then use these here and below.

Reply:
Done. And added more explanations in comments.